Vagrantfile

```diff
@@ -3,4 +3,14 @@
 
 Vagrant.configure("2") do |config|
   config.vm.box = "debian/bullseye64"
+
+  # Provider for Docker, necessary for Apple Silicon chips
+   config.vm.provider :docker do |docker, override|
+     override.vm.box = nil
+     docker.image = "rofrano/vagrant-provider:ubuntu"
```

Comment:
all our servers are on debian, maybe we should have it too?

Reply:
Sure. @martinratinaud can you help us identifying a suitable Debian image? 🙂